### PR TITLE
Feature/3642 remove references of js templates

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3642

#### Changes proposed in this Pull Request:
- Removed react and react native js template references

#### To Test:
- yarn start

### CODE REVIEW

File: src/docs/development/frameworks-web/react.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ The fastest way to start a new Brightlayer UI project is to use the [Brightlayer
 npx -p @brightlayer-ui/cli blui new react
 ```
 
-You will be asked to provide a name for your project, choose whether you would like to use TypeScript or JavaScript, and for TypeScript projects, you'll be asked if you want to use the Brightlayer UI eslint and prettier configurations files.
+You will be asked to provide a name for your project, you'll be asked if you want to use the Brightlayer UI eslint and prettier configurations files.
```
Comment (daileytj) [inline src/docs/development/frameworks-web/react.mdx:17]:
```suggestion
You will be asked to provide a name for your project and whether you want to use the Brightlayer UI eslint and prettier configuration files.
```

**Review by daileytj — CHANGES_REQUESTED** [review]
I would undo all of the changes in the src/docs/migration.mdx file as that should live as a historical record of the migration instructions from pxblue to BLUI. It doesn't look like we actually use that doc any longer so we may want to just remove it in the future. For now I would leave it as it was.

I only looked at the code changes soo far. I'll need to dig into the source code to see if there are any lingering mentions of these resources.

**Comment by daileytj** [comment]
- [ ] The [developer section of visualizations design patterns](https://blui-doc-it--pr608-feature-3642-remove-ycxcv4y7.web.app/patterns/visualizations#developers) should no longer reference high charts, chart js, and mapbox. 
- [ ] @joebochill should we drop support for victory native as well? (I'm not even sure which resources we have to support that). If so, remove that as well.
- [ ] In place of the items removed, add a blurb about DDS and links to their site and storybook (I'll add them here when I confirm what they are)
- [ ] Update the "Can I use other charting solutions..." answer on the [FAQ page](https://blui-doc-it--pr608-feature-3642-remove-ycxcv4y7.web.app/community/faq)

**Comment by bkarambe** [comment]
> * [x]  The [developer section of visualizations design patterns](https://blui-doc-it--pr608-feature-3642-remove-ycxcv4y7.web.app/patterns/visualizations#developers) should no longer reference high charts, chart js, and mapbox.
> * [ ]  @joebochill should we drop support for victory native as well? (I'm not even sure which resources we have to support that). If so, remove that as well.
> * [ ]  In place of the items removed, add a blurb about DDS and links to their site and storybook (I'll add them here when I confirm what they are)
> * [ ]  Update the "Can I use other charting solutions..." answer on the [FAQ page](https://blui-doc-it--pr608-feature-3642-remove-ycxcv4y7.web.app/community/faq)

In the answer of that FAQ, we are recommending chartjs and highcharts libraries. Those are not blui libraries. Don't we want to recommend those?

**Comment by joebochill** [comment]
Friendly reminder to target the `feature/mui5-migration` branch to make @daileytj's life easier when he picks up that story. Otherwise he's going to have to redo all of these changes.

**Comment by joebochill** [comment]
Please update the PR title since this has nothing to do with JS CLI templates

File: src/docs/patterns/visualizations.mdx [hunk 2]
```diff
@@ -122,18 +122,15 @@ There are not as many charting libraries available for React Native applications
 
 There are many libraries and services for data visualizations. Brightlayer UI currently provides themes / utilities for the following libraries:
 
-**Angular** and **React**:
+**React**:
 
--   Charts & Graphs
+-   Brightlayer Charts
```
Comment (joebochill) [inline src/docs/patterns/visualizations.mdx:127]: Combine all of this into one line like:

'- [Brightlayer Charts](link to docs) ([Storybook](link to storybook))'

Comment (joebochill) [reply src/docs/patterns/visualizations.mdx:127]: This section isn't really about explaining what these recommendations are, just linking to them. We will revamp this entire page in a future story to better steer users in that direction. For now we're just trying to remove references to deprecated stuff.

**Comment by joebochill** [comment]
As for victory native, we should leave that for now. We don't have any official resources related to it, it's just a recommendation. And DDS doesn't have a solution for RN that would replace it.

**Comment by joebochill** [comment]
Do you also want to include changes to address the icon repository restructuring in this PR too, or save that for a separate one?

**Comment by bkarambe** [comment]
> Do you also want to include changes to address the icon repository restructuring in this PR too, or save that for a separate one?

It would be better if we save it for later

**Comment by bkarambe** [comment]
Raised another PR against mui migration branch. https://github.com/brightlayer-ui/doc-it/pull/610

**Pull request closed without merging**